react-form-controls/my-app/src/RegistrationFormControlled.jsx

```diff
@@ -0,0 +1,35 @@
+import { useState } from 'react';
+
+export default function RegistrationFormControlled() {
+  const [userName, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+
+  function handleSubmit(e) {
+    e.preventDefault();
+    console.log('state:', { userName: `${userName}`, password: `${password}` });
+  }
+
+  return (
+    <form method="post" onSubmit={handleSubmit}>
```

Comment:
Since this is a Controlled form, you don't include the `method` attribute on a `<form>` element!  This is because React is controlling the form state and submission instead of typical HTML form submission.